packages/client/wallets/aa/src/blockchain/token/Tokens.ts

```diff
@@ -25,3 +29,54 @@ export type ERC20TransferType = { token: ERC2OEVMToken; amount: bigint };
 export type SFTTransferType = { token: SFTEVMToken; quantity: number };
 export type NFTTransferType = { token: NFTEVMToken };
 export type TransferType = ERC20TransferType | SFTTransferType | NFTTransferType;
+
+export function transferParams({
```

Comment:
nit: feels like there's a better file to put this in, even if we make one for it, as this doesn't have anything to do with the type definitions above

Reply:
I separated into `token.ts` & `transfer.ts`